### ISSUE TITLE
[1.14.4 + Server] Crash on startup

### ISSUE THREAD

**Issue by Wynadorn** [skyjay1/Nomadic-Tents#24]
Hey Skyjad1,

I'd like to submit a log for a crash on start-up on a new server.

Versions Used:
minecraft_server.1.14.4.jar
forge-1.14.4-28.1.1.jar
NomadicTents-1.14.4-10.0.1-beta.jar

Crashlog:
https://paste.ubuntu.com/p/5yFQ4YSSYJ/

First Exception:
````
00:04:25 [ERROR] [ne.mi.fm.ModList/LOADING] Encountered an exception during parallel processing
00:04:25 java.util.concurrent.ExecutionException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: Attempted to load class net/minecraft/client/renderer/color/ItemColors for invalid dist DEDICATED_SERVER
````

**Comment by Wynadorn** [comment]
Sorry, Closed, Duplicate #19

**Comment by skyjay1** [comment]
Actually, this does help. Now I know WHY ItemColors is causing a crash, and it's not just the other mod's fault. Now that Forge has a "stable" 1.14 build I will see if I can make any fixes. Thank you.
